### PR TITLE
add october post

### PR DESCRIPTION
**Pull request by josibake**

### CODE REVIEW

File: _posts/2023-10-11-october-bitdevs.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+title: "BitDevs Amsterdam Conference Edition, October 11th 2023"
+meetup: https://www.meetup.com/bitdevs-amsterdam/
+published: true
+---
+
+## BitDevs Amsterdam meeting 007!
+
+It's time for a special edition of BitDevs, _The BitDevs Conference Edition!_ We hope to have a lot of friends from out of town joining, so we'll be meeting at a different place and different time than usual! Please read the details below carefully!
+
+### Wednesday, October 11th 2023 @ 3PM
```
Comment (JWBurgers) [inline _posts/2023-10-11-october-bitdevs.md:13]: Let's make this @2PM.

File: _posts/2023-10-11-october-bitdevs.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+We will be meeting at the [MACA Amsterdam](https://www.maca.amsterdam/) in Amsterdam Noord:
+
+* Address: Ms. van Riemsdijkweg 61, 1033 RC Amsterdam
+* Google maps link: [https://maps.app.goo.gl/gZwCxM1EWxoAouU9A](https://maps.app.goo.gl/gZwCxM1EWxoAouU9A)
+
+<br />
+Doors open at 2:00PM and we will try to start promptly at 3PM. Light snacks will be provided and then we will join our friends for [De Bitcoin Meetup at Blast Galaxy](https://www.meetup.com/de-bitcoin-meetup/events/294584411/) for food and drinks (~10 min walk away, starting at 5PM).
```
Comment (JWBurgers) [inline _posts/2023-10-11-october-bitdevs.md:21]: Suggestion: "Light snacks will be provided. We will joint our friends for .....".

File: _posts/2023-10-11-october-bitdevs.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+Please RSVP (either through our meetup page or to *bitdevsamsteram at protonmail dot com*) as we are expecting a larger crowd than usual and space at our venue is limited to 100 people. We hope to see you at both for good discussion, good food, and good drinks with great people! Let's kick off the conference with a bang!
```
Comment (JWBurgers) [inline _posts/2023-10-11-october-bitdevs.md:23]: Clarification: "We hope to see you at both the BitDevs and the Blast Galaxy Bitcoin Meetup for good discussion..."

File: _posts/2023-10-11-october-bitdevs.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+As a reminder, the BitDevs rules are:
+
+1. No photos, videos, or recordings.
+1. [Chatham House Rule](https://en.wikipedia.org/wiki/Chatham_House_Rule): you may
```
Comment (JWBurgers) [inline _posts/2023-10-11-october-bitdevs.md:28]: "2. [Chatham House Rules].....

**Review by JWBurgers — CHANGES_REQUESTED** [review]

File: _posts/2023-10-11-october-bitdevs.md (hunk 1 above)
Comment (josibake) [reply _posts/2023-10-11-october-bitdevs.md:13]: I think this should still be the event start time, not the door open time. This is consistent with how we've done it previously.

**Pull request merged**